pkg/driver_atwinc15x0/Makefile.include

```diff
@@ -1 +1,4 @@
 INCLUDES += -I$(PKGDIRBASE)/driver_atwinc15x0/src
+
+# Disable 'ISO C forbids an empty translation unit [-Werror=pedantic]'
+CFLAGS += -Wno-pedantic
```

Comment:
Can this be set in `pkg/driver_atwinc15x0/Makefile` ? It would reduce the scope. As is it's applied to all built modules in the application.

Same comment for the semtech-loramac package.

Reply:
Heh, that was a convenient way to get around native gcc's next level of pedantry:

> error: format ‘`%p`’ expects argument of type ‘`void *`’, but argument 3 has type ‘`netdev_t *`’ {aka ‘`struct netdev *’`}